### DISCUSSION TITLE
Unable to create user

### DISCUSSION THREAD

**Discussion by xy-io** [johanohly/AirTrail#18] [General]
Hi, 
I've followed the Portainer instructions, and everything deploys as expected. However, I can't progress past the user creation screen. 

Are there any known issues which may cause this.

Thanks 

ps. Great project!

**Comment by johanohly** [comment]
It is indeed a known issue, related to CORS. Make sure you set the `ORIGIN` env var to the correct URL.

**Reply by xy-io** [reply]
Appreciate the quick reply and I'll check that now. Thanks

**Reply by xy-io** [reply]
That fixed it! Thanks again!

**Reply by johanohly** [reply]
> That fixed it! Thanks again!

Awesome, no worries!
